### ISSUE TITLE
Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".

### ISSUE THREAD

**Issue by luigifaticoso** [apptreesoftware/flutter_calendar#7]
On IOs everything works perfectly, on android it has some problems, it doesn't let me expand the calendar, i have to swipe three or four times and it works. Any fix?

```
Launching lib/main.dart on Android SDK built for x86 in debug mode...
Initializing gradle...
Resolving dependencies...
Running 'gradlew assembleDebug'...
Built build/app/outputs/apk/debug/app-debug.apk (30.6MB).
Installing build/app/outputs/apk/app.apk...
I/FlutterActivityDelegate( 5758): onResume setting current activity to this
Syncing files to device Android SDK built for x86...
D/        ( 5758): HostConnection::get() New Host Connection established 0x88a2d180, tid 5797
D/EGL_emulation( 5758): eglMakeCurrent: 0xa2650380: ver 3 0 (tinfo 0x8ddc8460)
I/flutter ( 5758): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter ( 5758): The following assertion was thrown during performLayout():
I/flutter ( 5758): SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".
I/flutter ( 5758): The maxPaintExtent is 229.38775510204079, but the paintExtent is 229.3877551020408. Maybe you have
I/flutter ( 5758): fallen prey to floating point rounding errors, and should explicitly apply the min() or max()
I/flutter ( 5758): functions, or the clamp() method, to the paintExtent? By definition, a sliver can't paint more than
I/flutter ( 5758): the maximum that it can paint!
I/flutter ( 5758): The RenderSliver that returned the offending geometry was:
I/flutter ( 5758):   RenderSliverGrid#9f188 relayoutBoundary=up32 NEEDS-LAYOUT NEEDS-PAINT
I/flutter ( 5758):   creator: SliverGrid ← SliverPadding ← ShrinkWrappingViewport ← _ScrollableScope ←
I/flutter ( 5758):   IgnorePointer-[GlobalKey#57fcf] ← Semantics ← Listener ← _GestureSemantics ←
I/flutter ( 5758):   RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#52ee1] ←
I/flutter ( 5758):   _ExcludableScrollSemantics-[GlobalKey#04cfd] ← RepaintBoundary ← CustomPaint ← ⋯
I/flutter ( 5758):   parentData: paintOffset=Offset(0.0, 0.0) (can use size)
I/flutter ( 5758):   constraints: SliverConstraints(AxisDirection.down, GrowthDirection.forward, ScrollDirection.idle,
I/flutter ( 5758):   scrollOffset: 0.0, remainingPaintExtent: Infinity, crossAxisExtent: 401.4, crossAxisDirection:
I/flutter ( 5758):   AxisDirection.right, viewportMainAxisExtent: Infinity)
I/flutter ( 5758):   geometry: SliverGeometry(scrollExtent: 229.4, paintExtent: 229.4, maxPaintExtent: 229.4,
I/flutter ( 5758):   hasVisualOverflow: true)
I/flutter ( 5758):   currently live children: 0 to 41
I/flutter ( 5758): 
I/flutter ( 5758): When the exception was thrown, this was the stack:
I/flutter ( 5758): #0      SliverGeometry.debugAssertIsValid.<anonymous closure>.verify (package:flutter/src/rendering/sliver.dart:601:9)
I/flutter ( 5758): #1      SliverGeometry.debugAssertIsValid.<anonymous closure> (package:flutter/src/rendering/sliver.dart:618:15)
I/flutter ( 5758): #2      SliverGeometry.debugAssertIsValid (package:flutter/src/rendering/sliver.dart:630:6)
I/flutter ( 5758): #3      RenderSliver.debugAssertDoesMeetConstraints (package:flutter/src/rendering/sliver.dart:984:21)
I/flutter ( 5758): #4      RenderObject.layout.<anonymous closure> (package:flutter/src/rendering/object.dart:1572:19)
I/flutter ( 5758): #5      RenderObject.layout (package:flutter/src/rendering/object.dart:1572:67)
I/flutter ( 5758): #6      RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:182:11)
I/flutter ( 5758): #7      RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #8      RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:285:13)
I/flutter ( 5758): #9      RenderShrinkWrappingViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1335:12)
I/flutter ( 5758): #10     RenderShrinkWrappingViewport.performLayout (package:flutter/src/rendering/viewport.dart:1298:20)
I/flutter ( 5758): #11     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #12     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #13     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #14     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #15     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #16     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #17     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #18     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #19     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #20     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #21     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #22     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #23     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #24     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #25     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #26     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #27     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #28     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #29     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #30     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #31     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #32     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #33     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #34     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #35     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #36     RenderStack.performLayout (package:flutter/src/rendering/stack.dart:514:15)
I/flutter ( 5758): #37     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #38     RenderAnimatedSize.performLayout (package:flutter/src/rendering/animated_size.dart:172:11)
I/flutter ( 5758): #39     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #40     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #41     _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1141:11)
I/flutter ( 5758): #42     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #43     RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:798:17)
I/flutter ( 5758): #44     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #45     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #46     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #47     RenderSliverList.performLayout.advance (package:flutter/src/rendering/sliver_list.dart:200:17)
I/flutter ( 5758): #48     RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:233:19)
I/flutter ( 5758): #49     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #50     RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:182:11)
I/flutter ( 5758): #51     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #52     RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:285:13)
I/flutter ( 5758): #53     RenderShrinkWrappingViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1335:12)
I/flutter ( 5758): #54     RenderShrinkWrappingViewport.performLayout (package:flutter/src/rendering/viewport.dart:1298:20)
I/flutter ( 5758): #55     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #56     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #57     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #58     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #59     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #60     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #61     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #62     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #63     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #64     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #65     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #66     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #67     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #68     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #69     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #70     _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:107:13)
I/flutter ( 5758): #71     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #72     RenderPadding.performLayout (package:flutter/src/rendering/shifted_box.dart:199:11)
I/flutter ( 5758): #73     RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
I/flutter ( 5758): #74     MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:141:11)
I/flutter ( 5758): #75     _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:399:7)
I/flutter ( 5758): #76     MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:211:7)
I/flutter ( 5758): #77     RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:355:14)
I/flutter ( 5758): #78     RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1445:7)
I/flutter ( 5758): #79     PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:709:18)
I/flutter ( 5758): #80     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:270:19)
I/flutter ( 5758): #81     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:627:13)
I/flutter ( 5758): #82     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:208:5)
I/flutter ( 5758): #83     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
I/flutter ( 5758): #84     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
I/flutter ( 5758): #85     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:842:5)
I/flutter ( 5758): #86     _invoke (dart:ui/hooks.dart:120:13)
I/flutter ( 5758): #87     _drawFrame (dart:ui/hooks.dart:109:3)
I/flutter ( 5758): 
I/flutter ( 5758): The following RenderObject was being processed when the exception was fired:
I/flutter ( 5758):   RenderSliverGrid#9f188 relayoutBoundary=up32 NEEDS-LAYOUT NEEDS-PAINT
I/flutter ( 5758):   creator: SliverGrid ← SliverPadding ← ShrinkWrappingViewport ← _ScrollableScope ←
I/flutter ( 5758):   IgnorePointer-[GlobalKey#57fcf] ← Semantics ← Listener ← _GestureSemantics ←
I/flutter ( 5758):   RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#52ee1] ←
I/flutter ( 5758):   _ExcludableScrollSemantics-[GlobalKey#04cfd] ← RepaintBoundary ← CustomPaint ← ⋯
I/flutter ( 5758):   parentData: paintOffset=Offset(0.0, 0.0) (can use size)
I/flutter ( 5758):   constraints: SliverConstraints(AxisDirection.down, GrowthDirection.forward, ScrollDirection.idle,
I/flutter ( 5758):   scrollOffset: 0.0, remainingPaintExtent: Infinity, crossAxisExtent: 401.4, crossAxisDirection:
I/flutter ( 5758):   AxisDirection.right, viewportMainAxisExtent: Infinity)
I/flutter ( 5758):   geometry: SliverGeometry(scrollExtent: 229.4, paintExtent: 229.4, maxPaintExtent: 229.4,
I/flutter ( 5758):   hasVisualOverflow: true)
I/flutter ( 5758):   currently live children: 0 to 41
I/flutter ( 5758): This RenderObject had the following descendants (showing up to depth 5):
I/flutter ( 5758):   RenderRepaintBoundary#bc741 NEEDS-PAINT
I/flutter ( 5758):     RenderDecoratedBox#ce32f NEEDS-PAINT
I/flutter ( 5758):       RenderSemanticsGestureHandler#6bfbe NEEDS-PAINT
I/flutter ( 5758):         RenderPointerListener#341d2 NEEDS-PAINT
I/flutter ( 5758):           RenderPositionedBox#0e59d NEEDS-PAINT
I/flutter ( 5758):   RenderRepaintBoundary#61d1a NEEDS-PAINT
I/flutter ( 5758):     RenderDecoratedBox#338b1 NEEDS-PAINT
I/flutter ( 5758):       RenderSemanticsGestureHandler#b44b3 NEEDS-PAINT
I/flutter ( 5758):         RenderPointerListener#e8289 NEEDS-PAINT
I/flutter ( 5758):           RenderPositionedBox#0ddc7 NEEDS-PAINT
I/flutter ( 5758):   RenderRepaintBoundary#c94c3 NEEDS-PAINT
I/flutter ( 5758):     RenderDecoratedBox#54e57 NEEDS-PAINT
I/flutter ( 5758):       RenderSemanticsGestureHandler#4a57f NEEDS-PAINT
I/flutter ( 5758):         RenderPointerListener#e974b NEEDS-PAINT
I/flutter ( 5758):           RenderPositionedBox#d7082 NEEDS-PAINT
I/flutter ( 5758):   RenderRepaintBoundary#64806 NEEDS-PAINT
I/flutter ( 5758):     RenderDecoratedBox#db9d7 NEEDS-PAINT
I/flutter ( 5758):       RenderSemanticsGestureHandler#bbf8f NEEDS-PAINT
I/flutter ( 5758):         RenderPointerListener#c7758 NEEDS-PAINT
I/flutter ( 5758):           RenderPositionedBox#a6a23 NEEDS-PAINT
I/flutter ( 5758):   RenderRepaintBoundary#85586 NEEDS-PAINT
I/flutter ( 5758):     RenderDecoratedBox#c9b6b NEEDS-PAINT
I/flutter ( 5758):       RenderSemanticsGestureHandler#e4049 NEEDS-PAINT
I/flutter ( 5758):         RenderPointerListener#d658b NEEDS-PAINT
I/flutter ( 5758):           RenderPositionedBox#146a4 NEEDS-PAINT
I/flutter ( 5758):   ...(descendants list truncated after 25 lines)
I/flutter ( 5758): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter ( 5758): Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".
I/chatty  ( 5758): uid=10084(com.yourcompany.stateful) 1.ui identical 11 lines
I/flutter ( 5758): Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".
I/zygote  ( 5758): Do partial code cache collection, code=30KB, data=24KB
I/zygote  ( 5758): After code cache collection, code=30KB, data=24KB
I/zygote  ( 5758): Increasing code cache capacity to 128KB
I/flutter ( 5758): Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".
I/chatty  ( 5758): uid=10084(com.yourcompany.stateful) 1.ui identical 11 lines
I/flutter ( 5758): Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".

```

**Comment by miniigo** [comment]
Hi luigifaticoso, I got the same exception. Did you find any fix? Please help. Thank you in advance!

Another exception was thrown: SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".

**Comment by ericwindmill** [comment]
This is fixed in the update going out in a couple minutes version (0.0.6)

**Comment by yathit** [comment]
I got the same issue

```
I/flutter ( 6604): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter ( 6604): The following assertion was thrown during performLayout():
I/flutter ( 6604): SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".
I/flutter ( 6604): The maxPaintExtent is 230.85714285714283, but the paintExtent is 230.85714285714286. Maybe you have
I/flutter ( 6604): fallen prey to floating point rounding errors, and should explicitly apply the min() or max()
I/flutter ( 6604): functions, or the clamp() method, to the paintExtent? By definition, a sliver can't paint more than
I/flutter ( 6604): the maximum that it can paint!
I/flutter ( 6604): The RenderSliver that returned the offending geometry was:
I/flutter ( 6604):   RenderSliverGrid#83fac relayoutBoundary=up22 NEEDS-LAYOUT NEEDS-PAINT
I/flutter ( 6604):   creator: SliverGrid ← SliverPadding ← ShrinkWrappingViewport ← _ScrollableScope ←
I/flutter ( 6604):   IgnorePointer-[GlobalKey#dbbb9] ← Semantics ← Listener ← _GestureSemantics ←
I/flutter ( 6604):   RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#1e4b8] ←
I/flutter ( 6604):   _ScrollSemantics-[GlobalKey#7599b] ← RepaintBoundary ← CustomPaint ← ⋯
I/flutter ( 6604):   parentData: paintOffset=Offset(0.0, 0.0) (can use size)
I/flutter ( 6604):   constraints: SliverConstraints(AxisDirection.down, GrowthDirection.forward, ScrollDirection.idle,
I/flutter ( 6604):   scrollOffset: 0.0, remainingPaintExtent: 624.0, crossAxisExtent: 404.0, crossAxisDirection:
I/flutter ( 6604):   AxisDirection.right, viewportMainAxisExtent: 624.0, remainingCacheExtent: 874.0 cacheOrigin: 0.0 )
I/flutter ( 6604):   geometry: SliverGeometry(scrollExtent: 230.9, paintExtent: 230.9, maxPaintExtent: 230.9,
I/flutter ( 6604):   hasVisualOverflow: true, cacheExtent: 230.9)
I/flutter ( 6604):   currently live children: 0 to 41
```

**Comment by yathit** [comment]
Sorry, it is fixed after 0.0.7 update.

**Comment by rakeshlanjewarwork** [comment]
i have the same issue in another pluginn, can you explain why it is happening

https://github.com/AbdulRahmanAlHamali/flutter_pagewise/issues/35#issue-405955903
